x-pack/plugins/infra/public/alerting/log_threshold/components/alert_details_app_section/types.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Rule } from '@kbn/alerting-plugin/common';
+import { TopAlert } from '@kbn/observability-plugin/public';
+import { AlertSummaryField } from '@kbn/observability-plugin/public';
+import { PartialRuleParams } from '../../../../../common/alerting/logs/log_threshold';
+
+export interface AlertDetailsAppSectionProps {
+  rule: Rule<PartialRuleParams>;
+  alert: TopAlert;
+  timeZone: string;
+  setAlertSummaryFields: React.Dispatch<React.SetStateAction<AlertSummaryField[] | undefined>>;
+}
```

Comment:
Are we not supposed to export the Props directly from the component file?

Reply:
Not sure if it is the same in the infra plugin.

Reply:
oh yes indeed, it might be different